src/mcts/search.cc

```diff
@@ -721,7 +721,9 @@ int SearchWorker::PrefetchIntoCache(Node* node, int budget) {
     return 1;
   }
 
-  // If it's a node in process of expansion or is terminal, don't prefetch it.
+  // n = 0 and n_in_flight_ > 0, that means the node is being extended.
+  if (node->GetN() == 0) return 0;
+  // The node is terminal; don't prefetch it.
   if (!node->HasChildren()) return 0;
```

Comment:
After some more thinking, I think this line as also better to change into
```cpp
  if (node->IsTerminal()) return 0;
```

Does the same, but kind of matches the comment.